modelci/types/ao/model_ao.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
```

Comment:
what is ao？

Reply:
Use python to execute this file, if made executable. Same as `#!/usr/bin/env bash`. 
These lines are generated by IDE.

Reply:
？

Reply:
What is `ao` in the file name?
@YuanmingLeee

Reply:
@YuanmingLeee  AO: Active Object?

Reply:
![image](https://user-images.githubusercontent.com/36268431/85227526-af323100-b410-11ea-8b16-e0e40d64f9da.png)
AO (application object). my bad, VO (view object) is more appropriate. 

From https://alitech-private.oss-cn-beijing.aliyuncs.com/1561031481870/Java-huashanxinban.pdf?Expires=1592754105&OSSAccessKeyId=LTAIqKGWQyF6Vd3W&Signature=I5xyksQpXbHNgDR%2B3YvDCT8sMyM%3D